r-package/grf/bindings/AnalysisToolsBindings.cpp

```diff
@@ -70,6 +70,11 @@ Eigen::SparseMatrix<double> compute_sample_weights(Rcpp::List forest_object,
   size_t num_samples = data->get_num_rows();
   size_t num_neighbors = train_data->get_num_rows();
 
+  // From http://eigen.tuxfamily.org/dox/group__TutorialSparse.html:
+  // Filling a sparse matrix effectively
+  typedef Eigen::Triplet<double> T;
```

Comment:
Sorry for the late review, I think we should avoid `typedef` as much as possible -- we don't really use it elsewhere, and it can make things harder to read. I think the `auto` keyword covers a lot of the cases where a `typedef` could help (although it doesn't apply to this example).

Reply:
This is corrected in #580